src/blueapi/worker/reworker.py

```diff
@@ -83,8 +87,26 @@ def __init__(
         self._stopping = Event()
         self._stopped = Event()
 
-    def submit_task(self, name: str, task: Task) -> None:
-        active_task = ActiveTask(name, task)
+    def clear_task(self, task_id: str) -> None:
+        del self._pending_tasks[task_id]
```

Comment:
Should this raise an error if the task doesn't exist? If it matters that the task doesn't exist, it could return a true/false value.

Reply:
I think it does matter since this will eventually be propagated up to the REST API and we want a DELETE operation to fail if the resource doesn't exist. Whether that should be an exception or return value, I have no strong feelings in this case. Have changed to return True or False.